src/devices/acurite.c

```diff
@@ -417,15 +417,24 @@ static int acurite_txr_callback(bitbuffer_t *bitbuf) {
 	    tempc = acurite_txr_getTemp(bb[4], bb[5]);
 	    tempf = celsius2fahrenheit(tempc);
 
-	    printf("%s Acurite tower sensor 0x%04X Ch %c: %3.1F C %3.1F F %d %% RH\n",
-		   time_str, sensor_id, channel, tempc, tempf, humidity);
+        data_acquired_handler(data_make("time","",DATA_STRING, time_str,
+                                        "model", "",DATA_STRING,"Acurite Tower Sensor",
+                                        "id", "Sensor ID", DATA_FORMAT, "%#X", DATA_INT, sensor_id,
+                                        "channel", "Channel", DATA_FORMAT, "%c", DATA_INT, channel,
+                                        "temperature_C", "Temp ℃", DATA_DOUBLE, tempc,
+                                        "temperature_F", "Temp ℉", DATA_DOUBLE, tempf,
```

Comment:
Only output the native type, that looks like to be Celsius. Sensor id's should be int.